libraries/AP_Scripting/applets/Mission_Controller.md

```diff
@@ -0,0 +1,7 @@
+# Mission_Controller LUA script
+
+This script allows the user to have different missions (in files named "mission.txt") in three subdirectories of the main scripting directory and load them based on the position of a switch with the auxiliary function of "302". This allows easy, at the field selection of missions, particulary useful for changing autoland/RTL_AUTOLAND directions based on prevailing wing conditions. It uses the same switch function as the Scripting_Controller.lua script selector and the two may be used together, if desired.
```

Comment:
Perhaps it should be clearer that this is not "switch" missions, it's "select" mission. As far as I can see it fires once when the autopilot comes up, and if the user decides they had the switch in the wrong position, they cannot just change the switch position. It looks like they are going to have to reboot to select a different mission.

Reply:
not necessarily....I use this with the scripting controller to change missions when I switch scripts....a script restart works the same as a reboot...but the name change is fine

Reply:
not true....it loads on boot, script restart, or directory change...can do so at anytime, even mid air...but reloading a mission while in auto resets it to mission item one

Reply:
Ok - then perhaps make that clearer in the instructions. My point is not how it works but that it's not entirely clear how it works (evidence: our discussion just now).

Reply:
it is very easy to make it load mission on the switch change....but then you get a log of spam if used with the scripting controller applet since it restarts scripting every switch change and you get two load of the same mission, once when the switch changed and once on the restart from scripting controller

Reply:
I updated the readme

Reply:
Yes I think that's clearer. What I said is true if you are not also using the script switcher. The dependancy on the script switcher for lua restart is important information. I think this is good.